sdk/communication/azure-communication-rooms/azure/communication/rooms/_rooms_client.py

```diff
@@ -111,20 +116,27 @@ def create_room(
         """
         create_room_request = {
             "validFrom": valid_from,
-            "validUntil": valid_until
+            "validUntil": valid_until,
+            "pstnDialOutEnabled": pstn_dial_out_enabled,
         }
         if participants:
             create_room_request["participants"] ={
                 p.communication_identifier.raw_id: {"role": p.role} for p in participants
             }
+        _SERIALIZER = Serializer()
```

Comment:
We shouldn't be using this object as ultimately it will be removed from the SDK - though it's not really blocking for now, this should probably be refactored at some point.

Reply:
We used this object as the values need to be serialized, and to avoid duplicated code I reused the existing function, but we can plan to remove that in a future iteration